diffkemp/simpll/Result.h

```diff
@@ -24,6 +24,22 @@
 
 using namespace llvm;
 
+/// Type for storing information about code location of an 'object'
+/// (eg. location of macro definition).
```

Comment:
Since functions are the most common example, let's have them in the comment.
```suggestion
/// (eg. location of a function or a macro definition).
```

Reply:
Thanks, fixed.